_component_design/chips.md

```diff
@@ -7,38 +7,93 @@ usage: >
   Chips represent a conceptual unit, such as a user, a device, a filter term, or a chunk of metadata and provide the user with the visual understanding of this conceptual unit as well as the ability to interact with them.
 preview-image: preview-images/chip.svg
 resource: true
-status: in-progress
-last-modified: 2017-08-17
+status: stable
+last-modified: 2017-11-16
 ---
 
 {% include toc.html %}
 
+## Introduction
+
 <div class="hxRow">
-{% column left:"hxCol" %}
-{% figure [caption:"Basic"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/chips.svg){:width="100%"}
-{% endfigure %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+
+Chips are best used with indicating a list of items that may be part of a different workflow. For example, adding or removing a list of requestors on a ticket.
+
+
+### Best practices
+
+- Avoid using chips in multiple locations on a page. 
+- Ensure the user can concretely identify the item represented by the chip.
+- Using the expanded version can help ensure the information needed by the user is visible.
+
 {% endcolumn %}
-{% column left:"hxCol" %}
-{% figure [caption:"With Icon"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/chips-icon.svg){:width="100%"}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"The chip component"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/chips/chips-hero.png){: width="1440px"}
 {% endfigure %}
 {% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Delete"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/chip-delete.svg){:width="100%"}
+
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Default chip
+
+The default chip is a single circle that contains a preview of the item the chip represents. The preview can consist of the the following; letters to indicate the name, an image to indicate a user, or a product icon to indicate the type of device.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Default chip specifications"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/chips/chips-default.png){: width="580px"}
 {% endfigure %}
 {% endcolumn %}
+
 </div>
+
 <div class="hxRow">
-{% column left:"hxCol-4" %}
-{% figure [caption:"Hover"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/chip-hover.svg){:width="100%"}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Expanded chip
```

Comment:
Remove the Expanded Chip and Interactions sections.  They are not a requirement for SDP, and it's a concept right now.  There's a Jira story for Ty to run a visual design pass on expanded chips.  after that we can add the content back in.

Reply:
Ok, I will note that @lfielder is building this out right now, but it is still a concept. I'll comment these sections out for the time being until they have interactions and spec finalized.